calendar.music

```diff
@@ -477,14 +495,17 @@
 11/27	Guillaume Dufay dies in Cambrai, 1474
 11/27	Arthur Honegger dies near Paris, 1955
 11/28	Jean-Baptiste Lully is born in Florence, 1632
+11/28	Maria Farantouri (pronounced Farandouri) is born in Athens, 1947
```

Comment:
I would refrain from pronounciation specifications, they generate new difficult issues:

- which names would deserves pronunciation specifications? For example, would Leonard Bernstein require such a specification? French names often are pronounced differently from how there are spelled: should we specify that the 't' at the end of Bizet should not be pronounced?

-  how do we specify a pronounciation? Unless we start using the phonetic alphabet, it is hard to do be clear. Since the calendar entries are in English it can easily be infered that pronounciations written as words in latin alphabet should refer as to how an English would read such a word, but is it so unambiguous? Would a Bristish, an American and an Australian read "Farandouri" in the same way? I am unsure and I am also unsure that one of them would really get it right (would they read 'r', corresponding to the greek 'ρ', properly?).

Of course, I understand that the issue here is simply specifying that the 't' should be pronounced 'd', but I would still avoid it.

Calendar files are written documents, we do not have to care about pronounciations.

Reply:
On second thought, I haven't included any on names using the Greek delta, so removing this one for consistency.